tests/unit/filters/test_filters.py

```diff
@@ -9,8 +10,11 @@
 from power_grid_model_io.filters import (
```

Comment:
There is no unit test for `get_winding`, `all_true`, `any_true`

Reply:
Noted. I'll add them.

Reply:
Done.